rebuild/lib/index.js

```diff
@@ -5,6 +5,30 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 const bindings = require("bindings")("blst_ts_addon");
 
+bindings.verify = function verify(msg, pk, sig) {
+  return bindings.aggregateVerify([msg], [pk], sig);
+};
+
+bindings.asyncVerify = function asyncVerify(msg, pk, sig) {
+  return bindings.asyncAggregateVerify([msg], [pk], sig);
+};
+
+bindings.fastAggregateVerify = function fastAggregateVerify(msg, pks, sig) {
+  try {
+    return bindings.aggregateVerify([msg], [bindings.aggregatePublicKeys(pks)], sig);
+  } catch {
+    return false;
+  }
+};
+
+bindings.asyncFastAggregateVerify = function asyncFastAggregateVerify(msg, pks, sig) {
+  try {
+    return bindings.asyncAggregateVerify([msg], [bindings.aggregatePublicKeys(pks)], sig);
```

Comment:
if this returns promise, this try catch will not catch the errors ... so you need to do `bindings.asyncAggregateVerify([msg], [bindings.aggregatePublicKeys(pks)], sig).catch(_e){return false}`

Reply:
Good catch!! That was copy/pasta from the sync one in the other version.  Updated!